repo/cray-mpich-gtl/package.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from spack.package import *
+import os.path
+import os
+import stat
+from spack_repo.builtin.packages.mpich.package import MpichEnvironmentModifications
+from spack.package import LinkTree
+
+class CrayMpichWrappers(MpichEnvironmentModifications, BundlePackage):
+
+    version("1.0.0")
+
+    depends_on("cray-mpich", type="build")
+    provides("mpi")
```

Comment:
```suggestion
    provides("mpi@3")
```
To match the underlying package.